File: ManagemateAPI/ManagemateAPI/Management/M_Receipt/Input_Objects/Add_Receipt_Data.cs
using ManagemateAPI.Management.M_Session.Input_Objects;

namespace ManagemateAPI.Management.M_Receipt.Input_Objects
{
    public class Add_Receipt_Data
    {

        public Session_Data session { get; set; }

        public bool in_out { get; set; }

        public long order_id_FK { get; set; }

        public string element { get; set; }

        public string transport { get; set; }

        public string comment { get; set; }

    }
}

File: Managemate/ManagemateAngular/src/app/shared/interfaces/API_Input_Models/Receipt_Models/Add_Receipt_Data.ts
import {Session_Data} from "../../API_Other_Models/Session_Models/SessionData";

export interface Add_Receipt_Data{
    session ?: Session_Data;

    creation_date ?: Date;
    in_out : boolean;
    reservation:boolean;
    order_id_FK : number;
    element : string;
    transport : string;
    // comment : string;
}